android/app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,7 @@
          In most cases you can leave this as-is, but you if you want to provide
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Does this require the permission for functionality? Can we still use the messaging if a users opts out of this permission?

Reply:
Location is only for the latitude/longitude tracking, everything else works without it

Reply:
I also noticed you have not requested the usage of the compass hardware? Android docs indicate that you require:
"
<manifest ... >
    <uses-feature android:name="android.hardware.sensor.compass"
                  android:required="true" />
    ...
</manifest>
"
Is this handled elsewhere?

Reply:
I'm not sure to be honest, I haven't had to give permission for it. I'll check the plugin docs to see if it handles it

Reply:
The example on the plugin doesn't use that permission. It looks like the location permission above covers it